Source/WebCore/workers/WorkerFontLoadRequest.cpp

```diff
@@ -98,7 +98,9 @@ RefPtr<Font> WorkerFontLoadRequest::createFont(const FontDescription& fontDescri
 {
     ASSERT(m_fontCustomPlatformData);
     ASSERT(m_context);
-    return Font::create(m_fontCustomPlatformData->fontPlatformData(fontDescription, syntheticBold, syntheticItalic, fontCreationContext), Font::Origin::Remote);
+    if (!m_fontCustomPlatformData)
+        return nullptr;
+    return Font::create(Ref { *m_fontCustomPlatformData }->fontPlatformData(fontDescription, syntheticBold, syntheticItalic, fontCreationContext), Font::Origin::Remote);
```

Comment:
This is not necessary. We should mark `m_fontCustomPlatformData` as `const` instead and use `lazyInitialize()` to initialize it. It never gets overwritten, just lazily initialized.